keras/backend/theano_backend.py

```diff
@@ -1507,6 +1507,10 @@ def conv2d(x, kernel, strides=(1, 1), border_mode='valid',
                                  input_shape=image_shape,
                                  filter_shape=filter_shape)
     else:
+        # T.nnet.conv2d uses **kwargs, so the filter_dilation parameter will be
+        # ignored by versions that do not support it
+        assert 'filter_dilation' in inspect.getargspec(T.nnet.conv2d).args, \
+            'conv2d with filter dilation requires Theano 0.9.0dev2 or newer.'
```

Comment:
Done. I used a ValueError because that's also used for other similar error messages.